project-rome-docs/objectivec-api/remotesystems.commanding/MCDLaunchUriProvider.md

```diff
@@ -14,22 +14,22 @@ This class manages the handling of a URI through the launching of an application
 
 ## Properties 
 ### supportedUriSchemes
-`@property(nonatomic, readonly, strong, nonnull) NSArray<NSString*>* supportedUriSchemes;`
+`@property(nonatomic, readonly, strong, nullable) NSArray<NSString*>* supportedUriSchemes;`
 
 An array of strings representing supported URI schemes.
 
 ## Methods
 
 ### onLaunchUriAsync
-`- (void)onLaunchUriAsync:(nonnull NSString*)uri
-         withFallbackUri:(nonnull NSString*)fallbackUri
-         preferredPackageIds:(nonnull NSArray<NSString*>*)preferredPackageIds
-              completion:(nonnull void (^)(BOOL, NSError* _Nullable))completionBlock;`
+```
+- (void)onLaunchUriAsync:(nonnull NSString*)uri
+         options:(nullable MCDRemoteLauncherOptions*)options
+              completion:(nonnull void (^)(BOOL, NSError* _Nullable))completionBlock;
+```
 
 This method is called when a remote device attempts to launch a URI on this device.
 
 #### Parameters 
 * `uri` The URI to launch.
-* `fallbackUri` The web-friendly URI to launch in the event that the primary URI fails. This should be a http or https web address or a file.
-* `preferredPackageIds` The list of IDs of preferred applications to handle this URI.
+* `options` The web-friendly URI to launch in the event that the primary URI fails. This should be a http or https web address or a file.
```

Comment:
>The web-friendly URI to launch in the event that the primary URI fails. This should be a http or https web address or a file. [](start = 12, length = 125)

Just say that it's a set of options for launching the URI. The fallback URI is only one of the possible options that can be set.